libs/auth-api-lib/src/lib/entities/models/delegation-scope.model.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Column,
+  DataType,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  PrimaryKey,
+  ForeignKey,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+import { Delegation } from './delegation.model'
+
+@Table({
+  tableName: 'delegation_scope',
+  indexes: [
+    {
+      fields: ['delegation_id, scope_name'],
```

Comment:
Primary keys are already indexed. This is not needed, and I'm not sure how the ORM will try to map to a index.

Reply:
I see this is done in the other models as well. Causes a confusion to the reader

Reply:
I have seen bugs when this is not done.. but I will need to test this without defining this.

Reply:
hmm yeah ok, interesting 🤔